bridges/snowbridge/primitives/inbound-queue/src/v2/converter.rs

```diff
@@ -96,7 +106,9 @@ where
 	ConvertAssetId: MaybeEquivalence<TokenId, Location>,
 	GatewayProxyAddress: Get<H160>,
 	EthereumUniversalLocation: Get<InteriorLocation>,
-	GlobalAssetHubLocation: Get<Location>,
+	AssetHubFromEthereum: Get<Location>,
+	AssetHubUniversalLocation: Get<InteriorLocation>,
+	AccountId: Into<[u8; 32]> + From<[u8; 32]> + Clone,
```

Comment:
nit:

the order of the many generics of this struct is random, could use some logical grouping or something

Reply:
It also seems that we introduced too many location representations, such as  `AssetHubFromEthereum`, `AssetHubUniversalLocation`, `EthereumUniversalLocation` and `EthereumNetwork`, I’d assume some of these refer to the same thing, just in different contexts. This could potentially be simplified by consolidating and using only a few of them.